templates/bake/Template/index.twig

```diff
@@ -39,7 +39,7 @@
                 <tr>
 {% for field in fields %}
 {% set isKey = false %}
-{% if associations.BelongsTo %}
+{% if associations.BelongsTo is defined %}
```

Comment:
Turning on `strict_variables` by default for Twig will potentially cause errors in user templates. In order to keep upgrade experience smooth for users we should consider turning is off for bake.

Reply:
They are only enabled in debug which is turned on for unit tests. They shouldn't experience it locally ... right?

Reply:
bake is always used locally with debug on.